docker-compose.yml

```diff
@@ -10,24 +10,19 @@ services:
         - postgres
         env_file: netbox.env
         volumes:
-        - netbox-nginx-config:/etc/netbox-nginx/
         - netbox-static-files:/opt/netbox/netbox/static
     nginx:
         image: nginx:1.11-alpine
         command: nginx -g 'daemon off;' -c /etc/netbox-nginx/nginx.conf
-        depends_on:
-        - netbox
```

Comment:
What's the intention behind removing the `depends_on`?

Reply:
I believe it's there to make sure nginx is restarted once netbox rebuilt and restarted.
This is no longer needed since the config for nginx is not in the netbox image.

Reply:
It's there, so that docker-compose starts the netbox container automatically when the nginx container is started. Because the nginx container on it's own is useless.

Reply:
That's really backwards in my opinion.
If something, netbox is dependent on nginx and not the other way around given the fact that ngnix would still be accessible and work without netbox.
But since the order in which they are started in doesn't matter i'd say there is no dependency either way. 
https://docs.docker.com/compose/compose-file/#depends_on